ckanext/data_qld/actions.py

```diff
@@ -447,6 +447,8 @@ def list_datarequests(original_action, context, data_dict):
         return result
 
     query = data_dict.get("q", "")
+    if not query:
+        return result
```

Comment:
No sort order?

Reply:
re-reviewed, this is 'adding' to the original calls requests so no need to sort a already sorted list.

Reply:
The original function can handle that.